src/app/flashcard/flashcard.component.scss

```diff
@@ -0,0 +1,55 @@
+/* flashcard.component.scss */
+@use '@angular/material' as mat;
+@use '@angular/material-experimental' as matx;
+
+@mixin theme($theme){
+.flashcard {
+    width: 500px;
+    height: 282px;
+    transform-style: preserve-3d;
+    transition: transform 0.5s;
+    background-color: mat.get-theme-color($theme, primary-container);
+    box-shadow: 4px 0 8px rgba(0, 0, 0, 0.2), -4px 0 4px rgba(0, 0, 0, 0.2), 0 4px 4px rgba(0, 0, 0, 0.2);
+    border-radius: 12px;
+  }
+  
+  .flashcard.flipped {
+    transform: rotateY(180deg);
+  }
+  
+  .front,
+  .back {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    backface-visibility: hidden;
+    display: grid;
+    grid-template-columns: 8% auto 8%;
+  }
+
+  .arrow {
+    margin: auto;
+    margin-top: 8px;
+    grid-column: 1 / 2;
+  }
+
+  .front-content{
```

Comment:
Can you combine the front-content styling and the back-content styling since they're the same code. This can be accomplished by a similar way you did it in lines 20-21. Then do individual style definition for the text formatting. This is to reduce repeated code.